### ISSUE TITLE
Clean up code. | Edit multiple routes. 

### ISSUE THREAD

**Issue by rdok** [PadawanTony/BusTracker#12]
1. Add validation request | EditRouteRequest
2. Extract only data we work with. | EditRouterExtractor
3. Use single responsibility principle for editing all routes. | 
4. Add flash messages. Why session messages are preferred:
   1. Store messages in session data until they are retrieved from any page.
   2. Pass message to flash array from any where in the project. 

> See https://github.com/PadawanTony/BusTracker/commit/7ddef438e62ac400ba283d2096daed00e0a9704a

**Comment by PadawanTony** [comment]
@rdok I added flash messages. Feel free to clean up the code.

**Comment by rdok** [comment]
@PadawanTony Yep. I'll add flash messages based on sessions rather than just static ones.

**Comment by PadawanTony** [comment]
ok!

When you do it, please explain why it is better.
